include/algorithms/math/abs_types.h

```diff
@@ -97,17 +97,27 @@ namespace interface1
 /**
  * <a name="DAAL-CLASS-ALGORITHMS__MATH__ABS__INPUT"></a>
  * \brief %Input objects for the absolute value function
+ * \DAAL_DEPRECATED
  */
 class DAAL_EXPORT Input : public daal::algorithms::Input
 {
 public:
-    /** Default constructor */
+    /**
+     * Default constructor
+     * \DAAL_DEPRECATED_USE{ Batch() }
```

Comment:
why do you recommend to use Batch instead of Input

Reply:
fixed